oat-sa/tao-core
# Feature/ADF-1417/Upgrade to Node 18

### Related to: [ADF-1417](https://oat-sa.atlassian.net/browse/ADF-1417)

### Summary

Update the dependencies in order to support further versions of Node

### Details

To properly support Node 18, the dev dependencies had to be updated and aligned.

BREAKING CHANGE:
- The package now requires Node.js 14.17+

**Note:** The lock file was generated with `npm@6` so it would also work seamlessly with Node 14.

### How to test
- make sure to have Node 18 installed (you can use [nvm.sh](https://github.com/nvm-sh/nvm) for managing multiple versions)
- install the package:
    ```
    cd views
    npm i
    cd build
    npm i
    npx grunt taosass
    npx grunt taobundle
    npx grunt connect:dev taotest
    ```
